src/components/View/index.ts

```diff
@@ -167,7 +167,7 @@ class View extends BridgeBase {
 
     // NOTE: We're handling the possible Title and children in the templates below
 
-    let theInnerTemplate = nothing;
+    let theInnerTemplate;
```

Comment:
I see that you're removing the "= nothing" in a few places. Is there a reason for doing that?

Reply:
Yes John, For the above variables laterwe are assigning html templates, getting compilation errors if we are assigning initial value as nothing for html templates variables.

Reply:
I did a little experimentation and the error is a TypeScript error. I'll try to find time to investigate a better solution but, for now, we'd be better off using code like:

let theInnerTemplate: any = nothing;

to get some benefit of the Lit "nothing" than removing it and replacing with a comment. So, please try changes like that wherever "nothing" used to be used to see if that resolves the compilation errors. Thanks!

Reply:
sure John